README.md

```diff
@@ -211,6 +211,22 @@ class InlineComponent < ViewComponent::Base
 end
 ```
 
+It is also possible to render variants inline by creating additional `call_` methods.
+
+```ruby
+class InlineVariantComponent < ViewComponent::Base
+  def call
+    link_to "Default", default_path
+  end
+
+  def call_variant
+    link_to "variant", variant_path
+  end
+end
+```
+
+Using a mixture of templates and inline rendering in a component is not supported (templates will be prioritized).
```

Comment:
Yeah I was thinking further about this myself since I original wrote this - it's probably possible with a little more work - on reflection I can see it could be useful - I'll take a look!